aboutme.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>More DOM Manipulation</title>
+</head>
+
+<body>
+
+  <h1>About Me</h1>
+
+  <div class="profilePicture"></div>
+
+  <ul>
+    <li class="listitem">Nickname: sumayah <span id="nickname"></span></li>
+    <li class="listitem">Favorite Animal:none <span id="favorite"></span></li>
+    <li class="listitem">Hometown:kharj <span id="hometown"></span></li>
+  </ul>
+
+  <div class="favoriteBooks"></div>
+  <SCript src="book.js"></script>

```

Comment:
the first two letters should be small cases. 
```suggestion
  <script src="book.js"></script>
```